Source/DafnyTestGeneration/TestMethod.cs

```diff
@@ -622,6 +644,40 @@ public class TestMethod {
       return new List<string>();
     }
 
+    /// <summary>  Return the test input as a list of lines of code </summary>
+    public List<string> TestInputConstructionLines() {
+      List<string> lines = new();
+
+      foreach (var line in ValueCreation) {
+        // TODO: nested tuples will break
+        if (line.type is UserDefinedType userDefinedType && (userDefinedType.Name.StartsWith("_System.Tuple") || userDefinedType.Name.StartsWith("_System._tuple"))) {
+          lines.Add($"var {line.id}: " +
+                    $"({string.Join(",", line.type.TypeArgs.ConvertAll(typ => typ.ToString()))}) " +
+                    $":= {line.value};");
+        } else {
+          lines.Add($"var {line.id} : {line.type} := {line.value};");
+          var subsetTypeCondition = DafnyInfo.GetTypeCondition(line.type, line.id);
+          if (subsetTypeCondition != null) {
+            lines.Add("expect " + Printer.ExprToString(subsetTypeCondition) +
+                      ", \"Test does not meet type constraints and should be removed\";");
+          }
+        }
+      }
+
+      // assignments necessary to set up the test case:
+      foreach (var assignment in Assignments) {
+        lines.Add($"{assignment.parentId}.{assignment.fieldName} := " +
+                  $"{assignment.childId};");
+      }
+      
+      string receiver = "";
+      if (!DafnyInfo.IsStatic(MethodName)) {
+        receiver = ArgValues[0];
+        ArgValues.RemoveAt(0);
+      }
+      return lines;
```

Comment:
Is there a reason to have this in `TestInputConstructionLines`?

Reply:
I think it can be removed. I may have included it unnecessarily when copying this code chunk from TestMethodLines.